component/src/main/java/org/exoplatform/rhmanagement/integration.notification/HRContractAnniversaryNotificationPlugin.java

```diff
@@ -79,37 +79,36 @@ public boolean isValid(NotificationContext ctx) {
   @Override
 
   protected NotificationInfo makeNotification(NotificationContext ctx) {
-
-    UserRHDataDTO obj = ctx.value(EMPLOYEE);
-    String notifType = ctx.value(NOTIF_TYPE);
-    DateFormat formatter = new SimpleDateFormat("E MMM dd HH:mm:ss Z yyyy");
-
-    Set<String> receivers = new HashSet<String>();
-
-
+    NotificationInfo notif = NotificationInfo.instance();
     try {
+      UserRHDataDTO obj = ctx.value(EMPLOYEE);
+      String notifType = ctx.value(NOTIF_TYPE);
+      DateFormat formatter = new SimpleDateFormat("E MMM dd HH:mm:ss Z yyyy");
+
+      Set<String> receivers = new HashSet<String>();
       for (User rh : Utils.getRhManagers()){
         receivers.add(rh.getUserName());
       }
+      String userId=obj.getUserId();
+      StringBuilder activityId = new StringBuilder(userId);
+      activityId.append("-").append(obj.getUserId());
+      return NotificationInfo.instance()
+
```

Comment:
You should have applied the changes on the notif object created and keep the last return. This one is not needed